lib/tasks/migrate_paper_trail.rake

```diff
@@ -0,0 +1,19 @@
+require "paper_trail/frameworks/active_record"
+
+desc "Migrate PaperTrail YAML columns to JSON"
+task migrate_paper_trail: :environment do
+  # https://github.com/paper-trail-gem/paper_trail/blob/v12.3.0/README.md#convert-existing-yaml-data-to-json
+  most_recent_record = PaperTrail::Version.last.id
+
+  PaperTrail::Version.where.not(old_object: nil).find_each do |version|
+    puts "Migrating object #{version.id} / #{most_recent_record}"
```

Comment:
This is a lot of output, but I found it useful to see how long this was going to take!